### DISCUSSION TITLE
is it possible to be able to be compatible with commonjs?

### DISCUSSION THREAD

**Discussion by zzz6519003** [denoland/deno#20734] [Q&A]
XD

**Comment by markthree** [comment]
As far as I know, the deno community is more focused on the ecmascript module specification than the other runtimes, so it's likely that they won't support the commonjs specification at the application level, and only the npm ecosystem should be able to do some conversions for compatibility.

**Comment by bartlomieju** [comment]
Do you have specific use case in mind that is currently not possible?

**Comment by EdamAme-x** [comment]
There is a high possibility that we will not support commonjs in the future.

If who want to move it, who need to operate the bundle.

Deno team saying it like this, so it won't be supported.
https://deno.com/blog/commonjs-is-hurting-javascript
